orchagent/fgnhgorch.cpp

```diff
@@ -656,6 +701,8 @@ bool FgNhgOrch::set_new_nhg_members(FGNextHopGroupEntry &syncd_fg_route_entry, F
     SWSS_LOG_ENTER();
 
     sai_status_t status;
+    bool is_warm_reboot = false;
+    auto nexthopsMap = m_recoveryMap.find(ipPrefix.to_string());
```

Comment:
There is an inactive_to_active_map which needs to be populated as part of warm reboot. This is populated to a different value then itself, if an entire bank is down.

Reply:
yes. add the following below
 // case nhps in bank are all down
                if (fgNhgEntry->nextHops[bank_nh_memb.ip_address] != i)
                {
                	syncd_fg_route_entry.inactive_to_active_map[i] = fgNhgEntry->nextHops[bank_nh_memb.ip_address];
                }